src/lib/rest/httpRequestSend.cpp

```diff
@@ -471,7 +471,7 @@ int httpRequestSendWithCurl
 
   lmTransactionEnd();
 
-  return 0;
+  return res == CURLE_OK ? 0 : -9;
```

Comment:
(I missed this detail)
Especially it should be mentioned in the function header with the other error codes